ToolsTest/Володимир/MSTest.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NUnit.Framework;
+using Xunit;
+using Assert = Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
+using nAssert = NUnit.Framework.Assert;
+using xAssert = Xunit.Assert;
+using Theory = Xunit.TheoryAttribute;
+using Tools;
+using Tools.Models;
+
+namespace ToolsTest.Володимир
+{
+    [TestClass]
+    public class MSTest
+    {
+        //TryReplaceText GetCapitalTitle HasText
+        [TestMethod]
+        public void TestGetCapitalTitle()
+        {
+            //arrange
+            Article article = new Article { Title = "same article" };
+
+            //act
+            string result = article.GetCapitalTitle();
+
+            // Assert
+            Assert.AreEqual("Same article", result);
```

Comment:
This test is failed.
```suggestion
            Assert.AreEqual("SAME ARTICLE", result);
```